packages/libs/error-stack/tests/snapshots/test_debug__full__multiline_ctx.snap

```diff
@@ -1,27 +0,0 @@
----
```

Comment:
Why this file was removed? Was it unused?

Reply:
Yes, this file was removed when we renamed everything from ctx to context.